firebase_admin/firestore.py

```diff
@@ -50,15 +51,22 @@ def client(app=None):
     .. _Firestore Client: https://googlecloudplatform.github.io/google-cloud-python/latest\
           /firestore/client.html
     """
-    fs_client = _utils.get_app_service(app, _FIRESTORE_ATTRIBUTE, _FirestoreClient.from_app)
+
+    options = {"database_id": database_id} if database_id else None
+
+    fs_client = _utils.get_app_service(app, options, _FIRESTORE_ATTRIBUTE, _FirestoreClient.from_app)
     return fs_client.get()
 
 
 class _FirestoreClient:
     """Holds a Google Cloud Firestore client instance."""
 
-    def __init__(self, credentials, project):
-        self._client = firestore.Client(credentials=credentials, project=project)
+    def __init__(self, credentials, project, database_id=None):
+        if database_id:
+            self._client = firestore.Client(credentials=credentials, project=project, database=database_id)
+        else:
+            self._client = firestore.Client(credentials=credentials, project=project)
+
```

Comment:
we should add some tests for this new functionality

Reply:
I added a test to cover when an explicit database id is given